tools/tof_lib_gen/generate_dpkg.bash

```diff
@@ -1,6 +1,20 @@
+# Generating dpkg for Crosby
 echo "Generating dpkg for Ubuntu 18.04 Amd64"
-sudo DOCKER_BUILDKIT=1 docker build --file ubuntu_18_04/Dockerfile --output out_ubuntu18 .
+sudo DOCKER_BUILDKIT=1 docker build --file ubuntu_18_04/crosby/Dockerfile --output crosby/out_ubuntu18 .
 echo "Generating dpkg for Ubuntu 20.04 Amd64"
-sudo DOCKER_BUILDKIT=1 docker build --file ubuntu_20_04/Dockerfile --output out_ubuntu20 .
+sudo DOCKER_BUILDKIT=1 docker build --file ubuntu_20_04/crosby/Dockerfile --output crosby/out_ubuntu20 .
 echo "Generating dpkg for Ubuntu 22.04 Amd64"
-sudo DOCKER_BUILDKIT=1 docker build --file ubuntu_22_04/Dockerfile --output out_ubuntu22 .
+sudo DOCKER_BUILDKIT=1 docker build --file ubuntu_22_04/crosby/Dockerfile --output crosby/out_ubuntu22 .
+
+# enabling read and write 
```

Comment:
I think you are also enabling 'execution' besides 'read' and 'write'

Reply:
will replace it with a+rx (only read andexecute rights are needed)